### DISCUSSION TITLE
no reply to ticket

### DISCUSSION THREAD

**Discussion by ghost** [community/community#142788] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Hello ,

this is the 2nd ticket i have opened , and no one reply !!! 
how i can reach the support to help me 

please this is urgent
first ticket was from 1 month
2nd was from 1 week

**Comment by moranenzo** [comment]
<p>Hi there!</p>

<p>Changing your GitHub username is definitely a big step! You can do this by going to your account settings. Just keep in mind that changing your username may affect your repositories and links, so be sure to check out GitHub's documentation on how to properly manage the transition. If you have any further questions, feel free to ask!</p>

<p>Good luck, and happy coding!</p>

**Reply by ghost** [reply]
Hello ,
i am not asking to change my GitHub username , 
who said that?

**Reply by moranenzo** [reply]
shit mb, can u give your other request's id ?

**Reply by ghost** [reply]
3056874
this is the request id
i was having account under my email ( safah222@gmail.com)
and it got deleted ! and i now created new account under the same email 
so i am asking where is my old account , why it got deleted !!?

**Reply by moranenzo** [reply]
I understand how frustrating it must be to lose access to your old account. Accounts on GitHub can be deleted for various reasons, such as prolonged inactivity or violations of community guidelines.

To find out more about your old account and why it got deleted, I recommend reaching out to GitHub support. They should be able to provide you with specific information and help you recover your account if possible.

Good luck mate!

**Reply by ghost** [reply]
thank you 
as i mentioned before , i have opened 2 tickets , and no one reply ! so i am asking how i can reach support team as they are not answering to the tickets

**Reply by moranenzo** [reply]
If i well understood, if you have the free version of github the only way to contact the support is by the community tickets ...
All you can do is try to use this link : https://support.github.com/
You can also try to raise a real ticket (to the staff i mean) [here](https://docs.github.com/en/enterprise-cloud@latest/support/contacting-github-support/creating-a-support-ticket)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @safah222123, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
